code/controllers/subsystems/explosives.dm

```diff
@@ -258,17 +258,17 @@ SUBSYSTEM_DEF(explosives)
 			power -= O.explosion_resistance
 
 	if (power >= GLOB.config.iterative_explosives_z_threshold)
-		if ((z_transfer & UP) && HasAbove(epicenter.z))
+		if ((z_transfer & UP) && SSmapping.multiz_levels[epicenter.z][Z_LEVEL_UP])
```

Comment:
This seems inconsistent.
Sometimes you use this to check if a z-level exists above.

Reply:
They aren't inconsistent:
One is the translation for GetAbove
The other for HasAbove

One returns a turf,
the other the zlevel

One takes a zlevel value (number),
the other an atom

I have not invented anything new, just translated what was already there in the equivalent procs that the zlevel manager understands